src/components/__snapshots__/App.test.js.snap

```diff
@@ -114,7 +109,6 @@ exports[`App should render correctly 1`] = `
           },
           "render": [Function],
           "useStyles": [Function],
-          Symbol(Symbol.hasInstance): [Function],
```

Comment:
Not a big deal but do we know why this changed?

Reply:
Not sure, I'll take a look at the Jest migration docs to see if there's anything